src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+
```

Comment:
I think that 2 blank lines is enough between import statement and the rest of the code